portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -52,10 +52,16 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
     int numberOfCommentsToDisplay = 0;
-    
+    List<String> allTags = new ArrayList<>();
+
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-      List<String> allTags = new ArrayList<>();
+      String displayTag = getWhichTagToDisplay(request);
+      System.out.println(displayTag);
```

Comment:
Either remove this print or make this more informative with context such as
"Display tag in DataSerlet.doGet(): 'sports'".